docs/tutorials/run-kinto.rst

```diff
@@ -73,8 +73,8 @@ Then install the package using the default configuration:
 ::
 
     pip install kinto
-    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini
-    pserve kinto.ini
+    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini.sample -O kinto.ini
+    kinto --ini kinto.ini start
```

Comment:
We could just put the sample in the egg and suggest to run `kinto start` only :)